pkg/util/removeall/removeall.go

```diff
@@ -25,14 +25,14 @@ import (
 	"k8s.io/kubernetes/pkg/util/mount"
 )
 
-// RemoveAllOneFilesystem removes path and any children it contains.
+// OneFilesystem removes path and any children it contains.
 // It removes everything it can but returns the first error
 // it encounters. If the path does not exist, RemoveAll
 // returns nil (no error).
 // It makes sure it does not cross mount boundary, i.e. it does *not* remove
 // files from another filesystems. Like 'rm -rf --one-file-system'.
 // It is copied from RemoveAll() sources, with IsLikelyNotMountPoint
-func RemoveAllOneFilesystem(mounter mount.Interface, path string) error {
+func OneFilesystem(mounter mount.Interface, path string) error {
```

Comment:
I'm not sold on this name and would like to hear input from other reviewers.  The fact that this does something makes me think it should not be a noun, but include a verb.  `RemoveFromOneFileSystem` or something. 

I'm also wondering if this single-method util package should instead merge with another package

Reply:
+1 to both of these ideas. In my mind, the latter idea, of moving the single-method util package to merge with another package is probably the most important. Changing the func name seems slightly less important after that (and the golint error will be removed).